### ISSUE TITLE
Do we update our Freecen2 Places?

### ISSUE THREAD

**Issue by Captainkirkdawson** [FreeUKGen/FreeCENMigration#1017]
You will recall that Freecen2 Places is based upon Freereg Places which was based on an extraction from the Gazetteer of British Place Names
(https://www.gazetteer.org.uk/).  EricB and other spent upwards of a year tuning the extraction to meet the Reg needs. This involved disabling several thousand entries of crossroads and small settlements. In additional it rationalized duplicate names and cross boundary issues..

@geoffj-FUG has noted that it has recently been updated.
“Whereas the Gazetteer previously contained only settlements, we have expanded its remit to cover landscape features too. It now includes over 8,000 islands, 37,000 hills and mountains, 97,000 woodlands/forests and 16,000 lakes, ponds and marshes. Not to mention beaches, bays, waterfalls, coastal headlands, mountain ranges and so on. If it’s in Great Britain, we want it in the Gazetteer!”

We’ve also added in the communities of Wales and the Civil Parishes of England. A new ‘Area’ type enables us to include broader areas of the country (e.g. the Fens, Wentloog, Islandshire) which have a geographical and cultural identity. A new ‘Heritage Site’ type enables us to include historic houses, castle, Roman remains, neolithic sites etc. We have always included the ridings of Yorkshire. They have now been joined by the Parts of Lincolnshire in a new ‘Division’ field.

Have a look at what is included in the new version at https://abcounties.com/news/gazetteer-gets-an-upgrade/

He asks **Do we need to upgrade before we go live with FreeCEN2? It will be too late once validators have started updating**.

**Comment by geoffj-FUG** [comment]
The reason that I ask is that validators spend a lot of time researching Places of Birth. They are often the less commonly quoted geographic features which seem to have been covered in the new Gazetteer. The Gazetteer also includes Stately Homes etc. Servants, in particular, quote these as their Place of Birth. 
It seems to me that a lot of the additions that Validators will inevitably have to add to the Place Names Database are already covered. The update Gazetteer represents a significant saving in effort.
Geoff

**Comment by Captainkirkdawson** [comment]
@geoffj-FUG It appears that the version available for download remains the 2014 version! I have contacted them to see if they are planning to update their download file.

**Comment by Captainkirkdawson** [comment]
@geoffj-FUG wrote;
Whilst our Gazetteer contains lots of British Place Names it does not contain Irish Place Names. 

I raise this not in the context of the Ireland censuses but in the context of Validation Places of Birth where the person enumerated was born in Ireland.

I have found this Open Data which may or may not help us. It covers Northern Ireland only. https://www.nidirect.gov.uk/articles/gazetteers

I haven’t been able to find the equivalent in Eire.

**Comment by richpomfret** [comment]
Moving to 'on ice' until we have the data.

**Comment by geoffj-FUG** [comment]
As this has been put on ice I have extracted a little over 300 Irish Place names from my FreeCEN1 PlaceSup file.

 

I will gradually add the lat/long etc over the next few weeks.

 

Geoff

**Comment by PatReynoldsFUG** [comment]
Can this be closed or is it blocked by anything?

**Comment by geoffj-FUG** [comment]
The Gazetteer was updated from Irish Townlands download some time ago.

This story can be closed

Geoff
